app/assets/stylesheets/ama_layout/layout_components/button-grouping.scss

```diff
@@ -35,7 +35,6 @@
 
   &__button{
     @extend .button;
-    margin-bottom: 0; //assuming the button lives within a container, which causes 2 sets of margins
```

Comment:
Does the bottom margin get zero out like the other margins now? I think the margin bottom is coming from the button tag on most button groupings currently.